TOC.md

```diff
@@ -131,11 +132,11 @@
         + [Wrong Index Solution](/wrong-index-solution.md)
         + [Distinct Optimization](/agg-distinct-optimization.md)
       + [Prepare Execution Plan Cache](/sql-prepare-plan-cache.md)
```

Comment:
Does this topic related to "execution plan" need to be put under "Understanding the Query Execution Plan"?

Rest LGTM

Reply:
I took a look. The current location looks correct to me (under second topic of SQL Optimization Process, but not strictly logical or physical optimization).

Reply:
Got it.